### ISSUE TITLE
CSS interoperability with text-align

### ISSUE THREAD

**Issue by dani31415** [mathml-refresh/mathml#86]
We are deprecating some MathML attributes in favor of the equivalent ones in CSS. In particular, the CSS style text-align seems a good substitute for denomalign and numalign (since the deprecation in #30). For the same reasons, it is also a good candidate to replace the [columnalign](https://www.w3.org/TR/REC-MathML/chap3_5.html#sec3.5.1)  attribute in tables.

For example,
```
<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mfrac>
        <mn style="text-align:right">1</mn>
        <mn>1000</mn>
    </mfrac>
</math>
```
with result
![Fraction with numerator alignment to the right](https://dani31415.github.io/public-images/mathml-refresh/issue-0086-01.png)

**Comment by NSoiffer** [comment]
Yes (about colalign in core). HTML tables used to support something similar
for col and colgroup, but they went away in HTML5 in favor of text-align in
CSS. We should probably start a separate issue for this...

**Comment by fred-wang** [comment]
As explained on #30,  CSS text-align has features that don't seem good for MathML (apply to inline-level content, not inherited). @rwlbuis's proposal on that issue seems more aligned with the current implementations. In any case, change like this are unlikely to happen until we clarify the CSS box models (display values, handling of border/margins/padding etc).

**Comment by fred-wang** [comment]
I don't think we want text-align for MathML as it is supposed to apply to inline boxes.

Instead, the CSS align module looks to have what is needed here:  https://www.w3.org/TR/css-align-3/#intro

**Comment by NSoiffer** [comment]
The align module looks promising in terms of features, but what's it's implementation status? caniuse doesn't even list it yet. Also, there has been only one small update in the last year. That could be good sign (it's close to stable) or a bad sign (no interest). Anyone know which way the wind is blowing?
